.claude/skills/storybook-testing/SKILL.md

```
@@ -0,0 +1,854 @@
+await userEvent.keyboard("{Enter}");
+await userEvent.keyboard("{Escape}");
+
+// Hover
+await userEvent.hover(element);
+await userEvent.unhover(element);
+
+// Select (for native select elements)
+await userEvent.selectOptions(select, "optionValue");
+```
+
+**IMPORTANT: Direct userEvent vs userEvent.setup()**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)